### PR TITLE
CLN:Fix flake8 issues on test_run.py

### PR DESCRIPTION
**Pull request by dorothykiz1**
xref #1042

### CODE REVIEW

**Comment by datapythonista** [comment]
`setup.cfg` is conflicting here after merging the other PR.

File: test/test_run.py [hunk 1]
```diff
@@ -82,8 +81,8 @@ def _test_run(range_spec, branches, expected_commits):
         expected = set(['machine.json'])
         for commit in expected_commits:
             for psver in tools.DUMMY2_VERSIONS:
-                expected.add('{0}-{1}-py{2}-asv_dummy_test_package_1-asv_dummy_test_package_2{3}.json'.format(
-                    commit[:8], tool_name, pyver, psver))
+                expected.add(f'{commit[:8]}-{tool_name}-py{pyver}-asv_dummy_'
+                             f'test_package_1-asv_dummy_test_package_2{psver}')
```
Comment (datapythonista) [inline test/test_run.py:85]: You areremoving the `.json` here.
```suggestion
                             f'test_package_1-asv_dummy_test_package_2{psver}.json')
```

Comment (dorothykiz1) [reply test/test_run.py]: Interesting how i had missed it thank you

**Comment by datapythonista** [comment]
Thanks @dorothykiz1

**Pull request merged**